CMakeLists.txt

```diff
@@ -37,10 +37,16 @@ include (CheckIncludeFiles)
 set(original_run_e2e_tests ${run_e2e_tests})
 set(original_run_int_tests ${run_int_tests})
 set(original_run_unittests ${run_unittests})
+set(original_run_perf_tests ${run_perf_tests})
 
 set(run_e2e_tests OFF)
 set(run_int_tests OFF)
 set(run_unittests OFF)
+set(run_perf_tests OFF)
+
+if ((NOT TARGET azure_c_build_tools) AND (EXISTS ${CMAKE_CURRENT_LIST_DIR}/deps/ctest/deps/azure-c-build-tools/CMakeLists.txt))
+        add_subdirectory(deps/ctest/deps/azure-c-build-tools)
```

Comment:
>deps/ctest [](start = 25, length = 10)

Oh I guess this should also be added as its own submodule (azure-c-build-tools), I forgot about that. Not sure if it's related to your failures

Reply:
That's probably the issue, ctest might have an older version of the build tools that doesn't have that cmake function that the build is complaining about:

```
CMake Error at deps/azure-c-logging/CMakeLists.txt:40 (set_default_build_options):
  Unknown CMake command "set_default_build_options".
```

---
In reply to: [434196783](https://github.com/Azure/umock-c/pull/166#discussion_r434196783) [](ancestors = 434196783)